src/client/pages/Checkout/CheckoutDetails/QuoteDetails/QuoteDetails.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { colorsV3 } from '@hedviginsurance/brand'
+import styled from '@emotion/styled'
+import { getQuoteDetails } from 'utils/getQuoteDetails'
+import { useTextKeys } from 'utils/textKeys'
+import { OfferQuote } from '../../../OfferNew/types'
+import { SubSection } from '../components/SubSection'
+
+const { gray900, gray300, gray700 } = colorsV3
+
+const Row = styled.div`
+  font-size: 0.874rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0.5rem 0;
+  line-height: 1;
+`
+
+const Label = styled.div`
+  color: ${gray900};
+`
+const Value = styled.div`
+  color: ${gray700};
+  text-align: right;
+`
+
+const HorizontalSpacer = styled.div`
+  width: 48px;
+`
+
+const Group = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 0.5rem 0;
+  border-bottom: 1px solid ${gray300};
```

Comment:
Maybe we could make a `Divider` component instead of adding `border`, since we also need the same kind of line on the [payment page](https://www.figma.com/file/GQ1b2GBtO3SxGW2YaH7qng/Purchase-Flow-Checkout?node-id=1082%3A10868). And then you render it conditionally based on `index` below, to skip it after the last array item. 🤔 What do you think?

Reply:
That sounds like a good idea! I'll give it a go :)